### PR TITLE
Email notifications: update copy, troubleshooting link

### PR DESCRIPTION
**Pull request by sohkai**

### CODE REVIEW

File: src/components/SignatureRequest.js [hunk 1]
```diff
@@ -36,7 +36,10 @@ const SignerRequest = React.memo(function SignerRequest({
 
   const { statusText, statusTextColor } = useMemo(() => {
     if (signingError) {
-      return { statusText: 'Signature failed', statusTextColor: theme.negative }
+      return {
```
Comment (sohkai) [inline src/components/SignatureRequest.js:39]: This is what prettier did for me, but maybe I have an older version?

File: src/components/SignatureRequest.js [hunk 2]
```diff
@@ -134,8 +137,11 @@ const SignerRequest = React.memo(function SignerRequest({
               margin-top: ${3 * GU}px;
             `}
           >
-            {/* TODO- Add the link once we have it */}
-            <ActionButtons onClick={() => {}} compactMode={compactMode}>
+            {/* TODO- Add the link once https://github.com/aragon/help/issues/31 is resolved */}
```
Comment (sohkai) [inline src/components/SignatureRequest.js:140]:
```suggestion
            {/* TODO - Update the link once https://github.com/aragon/help/issues/31 is resolved */}
```

File: src/components/SignatureRequest.js (hunk 1 above)
Comment (fabriziovigevani) [reply src/components/SignatureRequest.js:39]: That's the correct format

**Comment by rperez89** [comment]
Let's solve the conflicts and LGTM

**Pull request merged**